src/main/webapp/css/imageUpload.css

```diff
@@ -0,0 +1,21 @@
+/* Form is hidden by default. */
+.hidden {
+  display: none;
+}
+
+#locationField, #controls {
```

Comment:
I'm not super familiar with CSS organization strategies, but it might make more sense to separate these and just have a block for locationField and a block for controls, since right now the styling for locationField is split across two blocks.